config/test/test.config.js

```diff
@@ -42,16 +42,21 @@ const pkgMap = fs
 module.exports = {
     rootDir: path.join(__dirname, "../../"),
     transform: {
-        "^.+\\.jsx?$": "<rootDir>/config/test/test.transform.js",
+        "^.+\\.(j|t)sx?$": "<rootDir>/config/test/test.transform.js",
     },
     restoreMocks: true,
     resetMocks: true,
     testEnvironment: "jsdom",
-    testMatch: ["<rootDir>/**/*_test.js", "<rootDir>/**/*_test.jsx"],
+    testMatch: [
+        "<rootDir>/**/*.test.js",
+        "<rootDir>/**/*.test.jsx",
+        "<rootDir>/**/*.test.ts",
+        "<rootDir>/**/*.test.tsx",
+    ],
```

Comment:
I think we could just eliminate this entirely as this looks like it's exactly the same as the default: https://jestjs.io/docs/configuration#testmatch-arraystring

I like the idea of shorter configs that lean more on defaults.

Reply:
I didn't realize the default would just work.  Thanks for the suggestion, I'll make that change. 🙂

Reply:
I tried using the default, but it didn't work because we have non-test files in our `__tests__` folders.  This resulted in the following errors:
```
 FAIL  packages/perseus/src/interactive2/__tests__/movable-mock.ts
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (node_modules/@jest/core/build/TestScheduler.js:133:18)
      at node_modules/@jest/core/build/TestScheduler.js:254:19
      at node_modules/emittery/index.js:363:13
          at Array.map (<anonymous>)
      at Emittery.emit (node_modules/emittery/index.js:361:23)

 FAIL  packages/perseus/src/__tests__/test-items/image-item.ts
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (node_modules/@jest/core/build/TestScheduler.js:133:18)
      at node_modules/@jest/core/build/TestScheduler.js:254:19
      at node_modules/emittery/index.js:363:13
          at Array.map (<anonymous>)
      at Emittery.emit (node_modules/emittery/index.js:361:23)

 FAIL  packages/math-input/src/components/input/__tests__/test-math-wrapper.jsx
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (node_modules/@jest/core/build/TestScheduler.js:133:18)
      at node_modules/@jest/core/build/TestScheduler.js:254:19
      at node_modules/emittery/index.js:363:13
          at Array.map (<anonymous>)
      at Emittery.emit (node_modules/emittery/index.js:361:23)
```

Reply:
Oh, gotcha.  Oh well, as you had it then. :)